pkg/apis/v1/nodepool_validation_cel_test.go

```diff
@@ -64,15 +64,15 @@ var _ = Describe("CEL/Validation", func() {
 	})
 	Context("Disruption", func() {
 		It("should fail on negative expireAfter", func() {
```

Comment:
nit. Can we move these test outside the disruption block? since they are meant to be in NodePool Template Spec

Reply:
I was thinking they're fine as is, since it's about disruption. The other sections are just labels, taints, and requirements. WDYT?